### PR TITLE
fix: copywriting for membership instead of crowdfunding

### PR DESCRIPTION
**Pull request by sk-tori**
This is an ongoing effort. I will complete the whole scope (meta tags, etc.) in the upcoming PRs.
We will soon have a Notion page outlining the new terminology too.

### CODE REVIEW

File: containers/PageEditProject/containers/ModalGroup$CreateProject/components/ModalSubmit/index.tsx [hunk 1]
```diff
@@ -176,7 +176,7 @@ export default function ModalSubmit({
       closeOnDimmerClick={false}
       closeOnEscape={!busy}
     >
-      <Modal.Header content="Extra" />
+      <Modal.Header content="Submit Creator" />
```
Comment (sk-kitsune) [inline containers/PageEditProject/containers/ModalGroup$CreateProject/components/ModalSubmit/index.tsx:179]: "Create Page" seems to be more accurate.

From my understanding, we are changing "project" -> "page" and keep the term "creator" as-is.

File: containers/PageHome/containers/PanelProtocolStatistics/index.tsx [hunk 2]
```diff
@@ -60,7 +46,7 @@ export default function PanelProtocolStatistics({
                 {stats.numProjects}
               </Card.Content>
             )}
-            <Card.Legend>Total projects</Card.Legend>
+            <Card.Legend>Creators</Card.Legend>
```
Comment (sk-kitsune) [inline containers/PageHome/containers/PanelProtocolStatistics/index.tsx:49]: The terms "page" and "creator" are being used in unorganized way. I think we should make it clear when to use "page" and when to use "creator".

When a creator closes a page and creates a new page, there are:
- One creator
- Two pages: one closed, one open

File: containers/PageEditProject/containers/ModalGroup$CreateProject/components/ModalSubmit/index.tsx (hunk 1 above)
Comment (sk-tori) [reply containers/PageEditProject/containers/ModalGroup$CreateProject/components/ModalSubmit/index.tsx:179]: Page is an off-chain component of the creator. This transaction creates the whole package, on-chain sponsorship, etc. included.

File: containers/PageHome/containers/PanelProtocolStatistics/index.tsx (hunk 2 above)
Comment (sk-tori) [reply containers/PageHome/containers/PanelProtocolStatistics/index.tsx:49]: No, page is a component of a creator, so the number of creators always equate the number of pages, and that page is not meaningful at the platform level. The same wallet can create and close multiple creators.

Comment (sk-kitsune) [reply containers/PageHome/containers/PanelProtocolStatistics/index.tsx:49]: > We will soon have a Notion page outlining the new terminology too.

Definitely we will need this document.

Comment (sk-tori) [reply containers/PageHome/containers/PanelProtocolStatistics/index.tsx:49]: Yeah, I think the worst thing is that we think of individuals when we say creators, but creators are really projects:
- There are two co-hosts in the hottest Patreon creator: https://www.patreon.com/TrueCrimeObsessed.
- Big solo Youtubers have an editing team, marketing team, etc.
- Technically, a creator can be multi-signed or single-signed by one of many different owners, etc., on-chain.

"Projects" do not sell, so we must go with creators for people in this market to relate instantly.
About "page", it is the same with "profile", "studio", etc. They are destinations on the platform / web app for users to hop into, not a first-class concept at the platform level.

**Pull request merged**